client/rest/src/test/java/org/elasticsearch/client/documentation/RestClientDocumentation.java

```diff
@@ -93,8 +94,8 @@ public void testUsage() throws IOException, InterruptedException {
 
         //tag::rest-client-init
         RestClient restClient = RestClient.builder(
-                new HttpHost("localhost", 9200, "http"),
```

Comment:
Usually I prefer 8 spaces for this kind of thing but I switched the snippets over to 4 to make the snippets fit in the box more easily.